src/ms.tex

```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
 
 \secauthor{Aarya Patil}
 
-Summarize funding sources (Moore, NASA) and amounts and what this has been used
-for.
+% Summarize funding sources (Moore, NASA) and amounts and what this has been
+% used for.
+
+With the goal of establishing a sustainable financial model for the Astropy
+project, the Coordination Committee under the new governance charter (see \S
+\ref{sec:project-governance}) established a standing Finance committee.
+This committee overlooks the planning and disbursement of finances on behalf
+of the project. The Astropy project accepts funds from institutions as well as
+individuals through NUMFOCUS\footnote{NumFOCUS is a 501(c)(3) nonprofit that
+supports and promotes world-class, innovative, open source scientific computing.},
+the fiscal sponsor of the project. Previously, no direct financial support
```

Comment:
```suggestion
supports and promotes world-class, innovative, open source scientific computing.}.
Previously, no direct financial support
```

I know this is what NF says themselves, but I think "fiscal sponsor" is often prone to misinterpretation as though they are giving us money as opposed to being more like the "bank".  (this goes with another comment below about NF).